src/scss/components/ChevronBreadcrumb.scss

```diff
@@ -23,9 +23,18 @@
     align-items: center;
     z-index: 2;
     color: white;
+    white-space: nowrap;
+    text-overflow: ellipsis;
+    overflow: hidden;
+}
+
+.chevronDiv div, span, p {
+    white-space: nowrap;
+    text-overflow: ellipsis;
+    overflow: hidden;
 }
 
-.chevronDiv span, p {
+.ellipsisText {
```

Comment:
Could you use [`text-truncate` from Bootstrap](https://getbootstrap.com/docs/5.1/helpers/text-truncation/)?